### PR TITLE
[Modular] Adds Clown & Mime Planet to the Fax.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

image
Pink because this seems to be the clown colour (the stamp, for example.), same with the Mime.

## Why It's Good For The Game

People already fax to Clown Planet. This just means they're pressing the button that says the place they want it.

## Changelog

:cl: Cursour, dictated by the Joker (Jack Nicholson and Jared Leto versions) and Pogo.
add: One may now directly fax Clown Planet. Mime Planet too if you hate trees.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
The joke of mime planet being a silent name is funny but actually not very apparent in this regard, Make it actually send to "Mime Planet" please
Although Mime and Clown planet being real places you can fax too is a little bit too uh, less than immersive.

**Comment by theselfish** [comment]
> The joke of mime planet being a silent name is funny but actually not very apparent in this regard, Make it actually send to "Mime Planet" please Although Mime and Clown planet being real places you can fax too is a little bit too uh, less than immersive.

Immersion is subjective.

I don't even know if there's a Mime Planet. And, I think it being three dots is in theme.

**Comment by YakumoChen** [comment]
The average non-mime-enjoyer is not going to figure out what it is, is the problem

I'm sure there's better names you could give these, like the Mime Collective

**Comment by theselfish** [comment]
> The average non-mime-enjoyer is not going to figure out what it is, is the problem
> 
> I'm sure there's better names you could give these, like the Mime Collective

Very fair. As much as it pains me, I'll edit it.

File: modular_zubbers/code/modules/paperwork/fax.dm [hunk 1]
```diff
@@ -13,6 +13,8 @@
 		Soato = list(fax_name = "Soato", fax_id = "Soato", color = "orange", emag_needed = TRUE),
 		SolFed = list(fax_name = "SBI", fax_id = "SBI", color = "orange", emag_needed = FALSE),
 		Irs = list(fax_name = "IRS", fax_id = "IRS", color = "orange", emag_needed = FALSE),
+		Clowns = list(fax_name = "Clown Planet", fax_id = "Clowns", color = "pink", emag_needed = FALSE),
+		Mimes = list(fax_name = "...", fax_id = "Mimes", color = "white", emag_needed = FALSE),
```
Comment (theselfish) [inline modular_zubbers/code/modules/paperwork/fax.dm:17]:
```suggestion
		Mimes = list(fax_name = "La Pantomime Troupe", fax_id = "Mimes", color = "white", emag_needed = FALSE),
```

**Pull request merged**